packages/contracts/test/ScriptConfiguration.t.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Technically, we could use a forge-std remapping in our non-production contracts (scripts, tests, etc). We chose not too because allowing
+// that would increase the chance that we accidentally ship something that may not compile in the users project because we accidentially
+// relied on a remapping in the wrong file.
```

Comment:
> We chose not too because allowing that would increase the chance that we accidentally ship something that may not compile in the users project because we accidentially relied on a remapping in the wrong file.

I don't understand this sentence. Would you mind clarifying?

Reply:
This is copied from the same comment in `Sphinx.t.sol`, so it's unrelated to this PR. I just kept it b/c it's helpful to have this documented in multiple places. 

I do think it's worth adding a sentence to this comment in both places for a bit more clarity. It should be something like this:

We do not use remappings for forge-std because if we did, the user would need to define them in their remappings, or our library contracts would not work. Technically, we could use a forge-std remapping in our non-production contracts (scripts, tests, etc.), such as this file, without forcing the user to define a remapping. We chose not to because doing that would increase the chance of accidentally shipping something that may not compile in the user's project because we accidentally relied on a remapping in the wrong file.